packages/core/src/shared/clients/codecatalystClient.ts

```diff
@@ -231,13 +243,14 @@ class CodeCatalystClientInternal {
     private async call<T>(req: AWS.Request<T, AWS.AWSError>, silent: boolean, defaultVal?: T): Promise<T> {
         const log = this.log
         const bearerToken = (await this.connection.getToken()).accessToken
-        req.httpRequest.headers['Authorization'] = `Bearer ${bearerToken}`
+
```

Comment:
> 3\. Removed the redundant bearerToken header in `CodeCatalystClientInternal.call()` since the token is added during the signing of the request in the aws-cdk

do you mean "aws-sdk"? perhaps this change should be a separate commmit, it's a bit subtle.

Reply:
Yes, I meant "aws-sdk", and you're right, this change is a bit unrelated so I'll put it into a separate commit.

Reply:
Moved to https://github.com/aws/aws-toolkit-vscode/pull/4616